src/ui/localization/locale/en.json

```diff
@@ -0,0 +1,57 @@
+{
+  "periods.week": "{{amount}} week",
+  "periods.month": "{{amount}} month",
+  "periods.year": "{{amount}} year",
+  "periods.day": "{{amount}} day",
+  "periods.weekShort": "{{amount}}wk",
+  "periods.monthShort": "{{amount}}mo",
+  "periods.yearShort": "{{amount}}yr",
+  "periods.dayShort": "{{amount}}d",
+  "periods.lifetime": "lifetime",
+  "periods.weekPlural": "{{amount}} weeks",
+  "periods.monthPlural": "{{amount}} months",
+  "periods.yearPlural": "{{amount}} years",
+  "periods.dayPlural": "{{amount}} days",
+  "periods.weekFrequency": "weekly",
+  "periods.monthFrequency": "monthly",
+  "periods.yearFrequency": "yearly",
+  "periods.dayFrequency": "daily",
+  "periods.unknownFrequency": "unknown",
+  "periods.weekFrequencyPlural": "every {{amount}} weeks",
+  "periods.monthFrequencyPlural": "every {{amount}} months",
+  "periods.yearFrequencyPlural": "every {{amount}} years",
+  "periods.dayFrequencyPlural": "every {{amount}} days",
+  "state_present_offer.product_title": "{{productTitle}}",
+  "state_present_offer.product_description": "{{productDescription}}",
+  "state_present_offer.product_price": "{{productPrice}}",
```

Comment:
what's the benefit of having this 3 keys?

Reply:
That the customer can customize them, doing stuff like

```
Price: {{productPrice}}
```